src/server/components/annual-review/controller.ts

```diff
@@ -132,12 +139,18 @@ async function redirectToFormRunner(req: Request, res: Response, next: NextFunct
   const result = await findListItemByReference(listItemRef);
 
   if (!result) {
-    return next(new HttpException(404, "404", "The list item cannot be found"));
+    next(
+      new HttpException(
+        404,
+        "404",
+        `The list item with reference ${listItemRef} cannot be found when trying to redirect to form runner`
```

Comment:
I think this will end up rendering for the user "cannot be found when trying to redirect to form runner", could you double check please? we shouldn't expose implementation detail - they shouldn't know/care what the form runner does.